apps/yggdrasil_web/web/router.ex

```diff
@@ -10,6 +10,7 @@ defmodule YggdrasilWeb.Router do
   pipeline :guardian do
     plug Guardian.Plug.VerifyHeader
     plug Guardian.Plug.EnsureAuthenticated, handler: YggdrasilWeb.GuardianErrorHandler
+    plug Guardian.Plug.LoadResource
```

Comment:
This whole block with the addition of LoadResource I think replaces the need for [this line](https://github.com/voluspa/yggdrasil/blob/master/apps/yggdrasil_web/web/controllers/session_controller.ex#L16) in the session controller. 

That line called api_signin but since each request through this pipeline digests the auth header and effectively does the same thing I don't believe the sign in is needed anymore as after login the first request that needs auth with effectively sign the user on via this block and make the user available down stream of his call just as the sign in would. 

This also means that if you have a valid token you don't have to explicitly call sign in. I'm ok with this. Since tokens have an expiration and checked in each time anyway and we don't have a real session. 

Any objections? This means the user could be idle and their connection be cleaned up and later on within that tokens expiration call the API without logging in explicitly via the login API call. For example the user comes back and refreshes their list of characters the next day it would just work not ask them to login provided the token is valid.